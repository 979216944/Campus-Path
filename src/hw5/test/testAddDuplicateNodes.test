# Tests that adding duplicate nodes does not affect and corrupt the graph.

# Create graph
CreateGraph g

# Add nodes to g
AddNode g a
AddNode g b
AddNode g c

# Add Duplicate nodes to g
AddNode g a
AddNode g b
AddNode g c

# List nodes and the outgoing edges
ListNodes g