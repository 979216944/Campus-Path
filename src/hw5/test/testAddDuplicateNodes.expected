# Tests that adding duplicate nodes does not affect and corrupt the graph.

# Create graph
created graph g

# Add nodes to g
added node a to g
added node b to g
added node c to g

# Add Duplicate nodes to g
added node a to g
added node b to g
added node c to g

# List nodes and the outgoing edges
g contains: a b c